apps/oc_chef_wm/priv/dispatch.conf

```diff
@@ -131,6 +131,8 @@
 
 %% Cookbook artifact endpoints
 {["organizations", organization_id, "cookbook_artifacts", name, identifier], oc_chef_wm_named_cookbook_artifact_version, []}.
+{["organizations", organization_id, "cookbook_artifacts", name], oc_chef_wm_cookbook_artifacts, [{resource_args, single_artifact}]}.
```

Comment:
It makes sense that we're using the same module for both, but I think the naming throws me off.  

The usual usage is: 

`{org/:id/thing} -> oc_chef_wm_thing` <- POST to create a Thing, and GET to list the Things
`{org/:id/thing/:thingname} -> oc_chef_wm_named_thing` <- PUT/DELETE/GET  to update/delete/fetch a Thing

And then in cases where there is more to a Thing it makes sense to: 
`{org/:id/thing/:thingname/:subthing} -> oc_chef_wm_named_thing`

But in this case the pattern follows is a bit different.   oc_chef_wm_cookbook_artifacts is handling the named cookbook artifact.  I think that most of that is because we're PUTing to a named artifact to create it, so perhaps a simple name change of oc_chef_wm_cookbook_artifact to oc_chef_wm_named_cookbook_artifact? 

Alternatively a separate module that just calls through for whatever is shared between them - that's what we recently did for keys/named_keys, when there was shared functionality.  Something two modules ( oc_chef_wm_cookbook_artifacts and oc_chef_wm_named_cookbook_artifacts) but with the latter calling into the former as needed? 

It isn't too big a deal, but when working on existing code, there's kind of an assumed "well I'm working with a named instance of something, so I want the `_named_` module".[1]  Keeping consistency where we can makes it easier to reason about. 

[1] I'm going to writing all this up fairly soon... as more people get into this code, the lack of this kind of documentation is becoming more of a problem.  In that way it's been really cool having new people hacking on oc_erchef, as it highlights some things we need to do to make developing for it more accessible.